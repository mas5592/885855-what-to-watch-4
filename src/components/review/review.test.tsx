import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Review from './review';

const review = {
  id: 4563456345,
  user: {
    id: 1223,
    name: `Ivan Ivanov`,
  },
  rating: 8.0,
  comment: `This movie is very interesting`,
  date: `2020-11-10T17:10:01.455Z`,
};

describe(`Review`, () => {
  it(`Should render correctly`, () => {
    const tree = renderer
      .create(<Review
        filmReview={review}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
