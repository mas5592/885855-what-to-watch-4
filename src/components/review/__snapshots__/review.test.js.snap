// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review Should render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      This movie is very interesting
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Ivan Ivanov
      </cite>
      <time
        className="review__date"
        dateTime="2020-11-10"
      >
        November 10, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    8,0
  </div>
</div>
`;
